### ISSUE TITLE
Handhelds with Maglocks

### ISSUE THREAD

**Issue by Doperwt** [tsunamayo/Starship-EVO#3000]
### Short Description of the issue:

If you add a mag lock to a handheld weapon you can pick things up but what you pick up gets a strange location and when you release it it is dropped at a random position(possibly intersecting the ground or another entity.

side note, mag locks don't gain the weight from the thing they pick up. you could potentially use a big mag lock to pick up 5000ton as a person or tiny ship.

### Steps to reproduce the issue:

Put mag lock on handheld(add way to control it)

### Environment
- Starship EVO version: current & experimental (20w37a)
- Graphics Card: not relevant

### Additional information:
<3

**Comment by Gliese-832-c** [comment]
I'd say limit handheld maglocks to 250Kg or so, so you can carry around small cargo containers, both built out of bricks, and actual ones, once they get added, but you shouldn't be able to tug a shuttle or ship with a small handheld tool. Maybe that limit could only be active in survival mode, but removed in creative mode?

**Comment by Doperwt** [comment]
> 
> 
> I'd say limit handheld maglocks to 250Kg or so, so you can carry around small cargo containers, both built out of bricks, and actual ones, once they get added, but you shouldn't be able to tug a shuttle or ship with a small handheld tool. Maybe that limit could only be active in survival mode, but removed in creative mode?

about the weight I would suggest that you could give a character a certain amount of strength/thrusters and make that the factor for controlling mag locked things
as in 100 newton to push around a person weighing 100kg(with spacesuit for example) would be really heavily affected by having more weight to lug around. That 100 newton would then have to push the 100kg person + weight of mag lock gun + weight of item

**Comment by Gliese-832-c** [comment]
Yeah, that's a much better idea. That way, you could theoretically even pull/push ships, however, after a certain weight, it would just get so slow, that you don't bother with it anymore.

**Comment by Hecateus** [comment]
interestingly, the trick above can also be done with Shipyard bricks.  They stick to Bricks, which stick to Gun Handles.

**Comment by CurioInventorium** [comment]
My experience with using a maglock on a gun was that the object locked would completely disappear. Although I didn't check in F6 mode to see if it was elsewhere nearby.
I did a YouTube video demonstrating the Shipyard one though :)
